src/blocks/dynamic-separator/test/dynamic-separator.cypress.js

```diff
@@ -127,6 +127,7 @@ describe( 'Test CoBlocks Dynamic Seperator Block', function() {
 	/**
 	 * Test dynamic separator custom class
 	 */
+	// pointer-events: none does not allow to click in addCustomBlockClass function
```

Comment:
Feel free to remove the comment if you are not able to reproduce the failing test.

Reply:
I left a few comments in places where we might encounter these errors again.